package authentication;

service Authentication {
  rpc Auth(Login) returns (UserPK);
}

message LoginRequest {
  string username = 1;
  string password = 2;
}

message UserPk {
  int32 pk = 1;
}